io/plugins/eu.esdihumboldt.hale.io.appschema.test/src/eu/esdihumboldt/hale/io/geoserver/rest/NamespaceResourceTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018 wetransform GmbH
```

Comment:
@nmco Just noticed this: you might want to change the copyright in the new source files

Reply:
Copyright updated.